pkgs/development/python-modules/gym/default.nix

```diff
@@ -14,8 +14,10 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "pyglet>=1.2.0,<=1.3.2" "pyglet"
+      --replace "pyglet>=1.2.0,<=1.3.2" "pyglet" \
+      --replace "cloudpickle>=1.2.0,<1.4.0" "cloudpickle~=1.2"
   '';
+  # cloudpickle range has been expanded in package but not yet released
```

Comment:
For future PRs, I think it would be better to have the comment *before* the thing you're commenting on. At least that seems to be the standard. Not important enough to hold this up though.